libraries/AP_Motors/AP_MotorsHeli_Swash.h

```diff
@@ -39,6 +39,11 @@ class AP_MotorsHeli_Swash {
     // Get function output mask
     uint32_t get_output_mask() const;
 
+#if HAL_LOGGING_ENABLED
```

Comment:
Need to `#include <AP_Logger/AP_Logger.h>` to use this here. It works so I guess its getting it via one of the other includes.

Reply:
Fixed